### PR TITLE
Adding changes for passing Golie version information via header req

### LINKED ISSUE #25: send http header user agent
Sending over full golie version is preferable.

This may one day turn out to be beneficial for some IT/ops guys trying to analyze their traffic.

### PR DESCRIPTION
**Pull request by carlosmmatos**
fixes #25

### CODE REVIEW

File: pkg/rolie/clone.go [hunk 1]
```diff
@@ -98,6 +99,8 @@ func (f *fetcher) getRemoteResourceRaw(URI string) (io.ReadCloser, error) {
 	if err != nil {
 		return nil, err
 	}
+	// Send GolieVersion in Header
+	req.Header.Set("Golie-Version", version.Version)
```
Comment (isimluk) [inline pkg/rolie/clone.go]: What about sending standard http `User-agent` request header? https://en.wikipedia.org/wiki/User_agent

**Review by isimluk — COMMENTED** [review]
LGTM, except I believe sending standard user-agent may be beneficial as existing software (think of cdn analytics) already parses that.

File: pkg/rolie/clone.go (hunk 1 above)
Comment (carlosmmatos) [reply pkg/rolie/clone.go]: looking into it now

Comment (carlosmmatos) [reply pkg/rolie/clone.go]: Are we planning on modifying the default user-agent being sent already? I added some debug just to print out the HEADER and USER-AGENT being passed:
```
INFO[2020-07-10T09:20:59-04:00] Received GET request from 127.0.0.1:64472 for ROLIE Feed at ../examples/rolie/feed/gov.nist.nvd.cve.recent.xml 
INFO[2020-07-10T09:20:59-04:00] HEADER_DUMP::map[Accept:[application/json] Accept-Encoding:[gzip] Golie-Version:[v0.1.0] User-Agent:[Go-http-client/1.1]] 
INFO[2020-07-10T09:20:59-04:00] USERAGENT_DUMP::Go-http-client/1.1 
```

I just want to make sure I understand your take on the user-agent.. if you would like to add send a custom one that is fine, we can figure it out.

Comment (isimluk) [reply pkg/rolie/clone.go]: Correct. I would went ahead and rewrite default user agent.

It holds no information. As soon as you see `golie-xyz`, you can conclude user agent was ABC based on version (`go-http-client` in our case).

Comment (carlosmmatos) [reply pkg/rolie/clone.go]: No problem - what do you have in mind? Would you rather send Golie version information via user-agent?

Comply with spec:
`User-Agent: <product> / <product-version> <comment>`
---
`User-Agent: Golie / v0.1.0`

I think we chatted yesterday about platform information.. is that still on the table?

Comment (isimluk) [reply pkg/rolie/clone.go]: Thanks! `User-Agent: Golie / v0.1.0` is fine with me. But you can add as much details as you like.

**Review by isimluk — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**